ci-operator/jobs/openshift/oadp-operator/openshift-oadp-operator-master-periodics.yaml

```diff
@@ -18,7 +18,7 @@ periodics:
       channel: '#forum-mig-ci'
       job_states_to_report:
       - failure
-      report_template: Job {{.Spec.Job}} failed.
+      report_template: Job {{.Spec.Job}} failed. URL {{.Status.Url}}
```

Comment:
`URL` needs to be uppercase

Reply:
You can also make the links clickable bu using syntax like `<{{.Status.URL}}|View logs>`

You can grep the repo for other examples of how to set up the notifications, like https://github.com/openshift/release/blob/a461abca5d3e0a7a23bb9c049da9220088b1acfd/ci-operator/jobs/kata-containers/kata-containers/kata-containers-kata-containers-main-periodics.yaml#L23-L26